src/ArgbColor.h

```diff
@@ -0,0 +1,80 @@
+#ifndef BLACKHOLERAYTRACER_ARGBCOLOR_H
+#define BLACKHOLERAYTRACER_ARGBCOLOR_H
+
+#include "CGL/color.h"
+
+using namespace CGL;
+
+class ArgbColor : public Color{
+public:
+  float a = 1; // alpha channel
+  static const ArgbColor White;
+  static const ArgbColor Black;
+  static const ArgbColor Transparent;
+
+  ArgbColor( float a = 0, float r = 0, float g = 0, float b = 0 )
+        : a(a), Color(r,g,b) { }
```

Comment:
This might just be a syntax thing, but according to this [link](https://www.cprogramming.com/tutorial/initialization-lists-c++.html), the parent initializer should always come first (probably b/c the child class might want to use the class attributes initialized in the parent class)

```suggestion
        : Color(r, g, b), a(a) { }
```